src/activities/Elsa.Activities.Kafka/Activities/SendKafkaMessage/SendKafkaMessage.cs

```diff
@@ -22,10 +22,12 @@ namespace Elsa.Activities.Kafka.Activities.SendKafkaMessage
     public class SendKafkaMessage : Activity
     {
         private readonly IMessageSenderClientFactory _messageSenderClientFactory;
+        private readonly KafkaOptions _kafkaOptions;
 
-        public SendKafkaMessage(IMessageSenderClientFactory messageSenderClientFactory)
+        public SendKafkaMessage(IMessageSenderClientFactory messageSenderClientFactory, KafkaOptions kafkaOptions)
```

Comment:
Is the `KafkaOptions` type registered with DI somewhere? If not, I think this will cause an exception when trying to resolve the activity from the service container.

Reply:
KafkaOptions are already injected as a singleton and are currently used for default SASL credentials.